index.bs

```diff
@@ -585,22 +585,29 @@ dictionary MLOperandDescriptor {
 </script>
 
 ## MLOperand ## {#api-mloperand}
+
+The {{MLOperand}} interface represents data that flows within the computational graph. See [[#programming-model]].
+
 <script type=idl>
 [SecureContext, Exposed=(Window, DedicatedWorker)]
 interface MLOperand {};
 </script>
 
 ## MLOperator ## {#api-mloperator}
 
-The {{MLOperator}} interface defines a type of operation such as the various types of activation function used to create other operations such as [[#api-mlgraphbuilder-conv2d]] or [[#api-mlgraphbuilder-batchnorm]].
+Objects implementing the {{MLOperator}} interface represent activation function types.
 
 <script type=idl>
 [SecureContext, Exposed=(Window, DedicatedWorker)]
 interface MLOperator {};
 </script>
 
 <div class="note">
-The implementation of this interface can simply be a struct that holds a string type of the operation along with other properties needed. The actual creation of the operation e.g. a [[#api-mlgraphbuilder-sigmoid]] or [[#api-mlgraphbuilder-relu]] can then be deferred until when the rest of the graph is ready to connect with it such as during the construction of [[#api-mlgraphbuilder-conv2d]] for example.
+These activations function types are used to create other operations such as [[#api-mlgraphbuilder-conv2d]] or [[#api-mlgraphbuilder-batchnorm]].
```

Comment:
We can say something like "one such use of this interface is for when an activation function is fused into another operation such as convolution or batchnorm during a graph construction session.."

Reply:
Thanks, updated per your suggestion.